client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
```

Comment:
I was unsure where to add it at the time but we covered that during our 
conversation this morning.

On 06/02/2015 06:01 PM, lifehackett wrote:
>
> In client/src/js/components/map.jsx 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8#discussion_r31580226>:
>
> > @@ -39,7 +39,17 @@ var Map = React.createClass({
> >        }
> >      ).addTo(map);
> >
> > -    L.geoJson(items).addTo(map);
> > +    var marker = new
> > +    L.Icon({
> > +      iconUrl: "images/marker-icon.png"
>
> Did you forget to add this image?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8/files#r31580226>.
>